common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -400,6 +400,12 @@
     ],
     "sqlState" : "58030"
   },
+  "CANNOT_SAVE_INTERVAL_TO_EXTERNAL_STORAGE" : {
```

Comment:
I wonder cannot you re-use `UNSUPPORTED_DATA_TYPE_FOR_DATASOURCE`?

Reply:
Sounds good. Just updated